domain/model/service/service.go

```diff
@@ -144,6 +148,20 @@ func (s *Service) DeleteModel(
 	return s.st.Delete(ctx, uuid)
 }
 
+// ModelList returns a list of all model UUIDs.
+// This only includes active models. Either a model is within the model manager
```

Comment:
Given we use "alive" below, it would be worth clarifying what "active" means. What determines whether a model uuid is in the result. If it's not going to return all models, should the api therefore be called "GetActiveModels" or "ActiveModelList" or whatever. I can imagine we would end up needing a "ModeList" later which really does return them all.
Also, IMO we should try and start service method names with a verb.

Reply:
Have updated the wording. I will review it when we get back to alive state.